src/pal/inc/rt/sal.h

```diff
@@ -2604,6 +2604,7 @@ extern "C" {
 #ifndef PAL_STDCPP_COMPAT
     #define __null
     #define __notnull
+    #define __deref
```

Comment:
I'm not sure if I should have done it for GCC only or not. I guess we'll see the build results.